app/services/usps_in_person_proofing/transliterable_validator_helper.rb

```diff
@@ -0,0 +1,88 @@
+module UspsInPersonProofing
+  class TransliterableValidatorHelper
+    SUPPORTED_FIELDS = [
+      :first_name,
+      :last_name,
+      :address1,
+      :address2,
+      :city,
+    ].freeze
+
+    def initialize(transliterator = Transliterator.new)
+      @transliterator = transliterator
+    end
+
+    # Validate that transliteratable fields from the state ID form
+    # can be transliterated to contain acceptable characters.
+    #
+    # @param [Hash] fields Name/value pairs of fields to validate
+    # @option fields [String,#to_s] :first_name
+    # @option fields [String,#to_s] :last_name
+    # @option fields [String,#to_s] :address1
+    # @option fields [String,#to_s] :address2
+    # @option fields [String,#to_s] :city
+    # @return [Hash,nil] Field names mapped to error messages, or nil if there were no issues
+    def validate(fields)
+      validation_result = {}
```

Comment:
I think if this class was combined with the validator class, we could take advantave of the built-in `ActiveRecord::Errors` instance and just `.add` there instead of having to pass pack things into a hash and then unpack them

Reply:
I'd like to see your response to [my prior comment](https://github.com/18F/identity-idp/pull/7938#discussion_r1128851186) before moving forward with this.

Reply:
The classes have been combined per our other conversation.